LICENSE

```diff
@@ -2110,6 +2334,255 @@ do so.  If you do not wish to do so, delete this exception statement
 from your version.
 
 
+---
+com.sun.xml.bind:jaxb-xjc
```

Comment:
Just move this like below line 1352, if the license is exactly the same.

Reply:
To me it doesn't look the same, on line 2371 there is a mention `Copyright (c) 2004 Kohsuke Kawaguchi` that is not present in `jaxb-core` license.

Reply:
ok